Makefile

```diff
@@ -236,6 +241,16 @@ $(RELEASE_DIR_BIN)/%:
 ###############################################################################
 # Utilities
 ###############################################################################
+helm: bin/helm
```

Comment:
Do you need this alias? 

Personally, I'd get rid of it and just use `bin/helm`